src/AvaloniaEdit/Search/SearchPanel.xaml

```diff
@@ -37,8 +37,17 @@
                 VerticalAlignment="Top" TextBlock.FontFamily="Segoi UI" TextBlock.FontSize="10">
                     <Grid ColumnDefinitions="Auto,Auto,Auto"
                 RowDefinitions="Auto,Auto">
+                        <Popup Name="PART_MessageView"
+                               Classes="ToolTip"
+                               PlacementMode="Bottom"
+                               StaysOpen="True"
+                               Focusable="False"
+                               IsOpen="False"
+                               ObeyScreenEdges="True">
+                            <ContentControl>Message Here</ContentControl>
```

Comment:
I think a `ContentPresenter` is good enough here. Also, add a named PART for the presenter, right now the code assumes a specific visual tree.

Reply:
Do your you mean use `ContentPresenter` instead of `ContentControl`?

Reply:
Yes